os/other-settings.md

```diff
@@ -33,10 +33,31 @@ coreos:
       command: restart
 ```
 
+### Loading Kernel Modules with Options
```

Comment:
I would argue this this. Linux world knows module options, not module configuration.